### PR TITLE
Added links and front page blog

### PR DESCRIPTION
**Pull request by Foxboron**
Signed-off-by: Morten Linderud <morten@linderud.pw>

### CODE REVIEW

File: content/pages/report.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ date: 2019-10-06T14:56:20+02:00
 
 ![Group Photo!](/images/conf/groupphoto.jpg)
 
+[Recordings](https://static.conf.archlinux.org/archconf2019/recordings/)
+
+Licensed under [CC-BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) by the Arch Linux team
+
+[Pictures](https://static.conf.archlinux.org/archconf2019/pictures/) [(Cloud Link)](https://www.jottacloud.com/p/foxboron/_3032773dd72a47ddb602c724685e35f1/thumbs)
```
Comment (dvzrv) [inline content/pages/report.md]: Is there an easy way to create a gallery of sorts?

File: content/pages/report.md [hunk 2]
```diff
@@ -5,6 +5,14 @@ date: 2019-10-06T14:56:20+02:00
...
+
+Licensed under [CC-BY-NC-ND 3.0](https://creativecommons.org/licenses/by-nc-nd/3.0/) by the Arch Linux team
```
Comment (dvzrv) [inline content/pages/report.md]: Why is it 3.0 here?

File: content/posts/arch_conf_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Arch Conf 2019 Report"
+date: 2019-11-13T20:50:45+01:00
+---
+
+Hey,
+
+During the 5th and 6th of October, 21 team members attending the very first
+internal Arch Conf. We spent 2 days at Native Instruments in Berlin having
+workshops, discussions and hack sessions together. Even managed to get into, and
```
Comment (dvzrv) [inline content/posts/arch_conf_report.md]: `We even...`

File: content/posts/arch_conf_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Arch Conf 2019 Report"
+date: 2019-11-13T20:50:45+01:00
+---
+
+Hey,
```
Comment (dvzrv) [inline content/posts/arch_conf_report.md]: Would also work without

File: content/posts/arch_conf_report.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+During the 5th and 6th of October, 21 team members attending the very first
+internal Arch Conf. We spent 2 days at Native Instruments in Berlin having
+workshops, discussions and hack sessions together. Even managed to get into, and
+escape, an escape room!
+
+It was a great and productive weekend which we hope will continue. Hopefully we
```
Comment (dvzrv) [inline content/posts/arch_conf_report.md]: `which we hope will continue in the next years.`

File: content/posts/arch_conf_report.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+![Group Photo!](/images/conf/groupphoto.jpg)
+
+[Recordings](https://static.conf.archlinux.org/archconf2019/recordings/)
+
+Licensed under [CC-BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) by the Arch Linux team
+
+[Pictures](https://static.conf.archlinux.org/archconf2019/pictures/) [(Cloud Link)](https://www.jottacloud.com/p/foxboron/_3032773dd72a47ddb602c724685e35f1/thumbs)
+
+Licensed under [CC-BY-NC-ND 3.0](https://creativecommons.org/licenses/by-nc-nd/3.0/) by the Arch Linux team
```
Comment (dvzrv) [inline content/posts/arch_conf_report.md]: Why is it 3.0 here?

File: content/pages/report.md (hunk 1 above)
Comment (Foxboron) [reply content/pages/report.md]: Not without additional software. Which is why i added my gallery for the sake of it now.

File: content/pages/report.md (hunk 2 above)
Comment (Foxboron) [reply content/pages/report.md]: I tried finding 4.0 but couldn't. Guess i typed wrong! I'll fix

File: content/posts/arch_conf_report.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Arch Conf 2019 Report"
+date: 2019-11-13T20:50:45+01:00
+---
+
+During the 5th and 6th of October, 21 team members attending the very first
```
Comment (jelly) [inline content/posts/arch_conf_report.md]: attended?

**Pull request merged**